presidio-analyzer/presidio_analyzer/predefined_recognizers/text_analytics_dal.py

```diff
@@ -0,0 +1,58 @@
+import http
+import os
+
+
+class TextAnalyticsDal:
+    def __init__(self, logger, tolerate_errors=True):
+        self.logger = logger
+        self.tolerate_errors = tolerate_errors
+        self.endpoint = os.environ.get('TEXT_ANALYTICS_ENDPOINT')
```

Comment:
Maybe we should disable it if the different environment variables are not set. This way it wouldn't work if a user does not have a TA service. We can throw an error when setting up the TextAnalyticsDal that TextAnalytics requires a b c to work, and it will always return an empty list if it doesn't hav these set up